src/cf-forms/src/molecules/input-with-btn.less

```diff
@@ -0,0 +1,49 @@
+//
+// Input with button
+//
+
+.m-input-with-btn {
+
+    .respond-to-min( 480px, {
+        .grid_nested-col-group();
+    });
+
+    &_input {
+        margin-bottom: unit( 15px / @base-font-size-px, em );
+        .respond-to-min( 480px, {
+            .grid_column(9);
+            border-right-width: 0;
```

Comment:
If we have to overwrite grid styles maybe we shouldn't use the grid? This feels brittle and bound to cause an issue at some point.